docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -140,100 +128,275 @@ You install YugabyteDB Anywhere on a Kubernetes cluster as follows:
+
+```yaml
+# yba-values.yaml
+zoneAffinity:
+  - us-west1-a
+  - us-west1-b
+```
+
+#### tolerations
+
+Kubernetes nodes could have taints that repel pods from being placed on it. Only pods with a toleration for the same taint are permitted. For more information, see 
+[Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/).
+
+For example, if some of the nodes in your Kubernetes cluster are earmarked for experimentation and have a taint `dedicated=experimental:NoSchedule`, only pods with the matching toleration will be allowed; other pods will be prevented from being placed on these nodes.
+
+```yaml
+# yba-values.yaml
 tolerations:
 - key: "dedicated"
   operator: "Equal"
   value: "experimental"
   effect: "NoSchedule"
 ```
 
+{{< note title="Scheduling the pods on dedicated nodes" >}}
+Tolerations don't guarantee scheduling on the tainted nodes. To make sure the YugabyteDB Anywhere pods use a dedicated set of nodes, you need to use [nodeSelector](#nodeselector) along with taints and tolerations to repel other pods.
+{{< /note >}}
+
+### Modify resources
+
+You can modify the resource requests and limits set for the various components of YugabyteDB Anywhere, these include CPU and memory resources. See Kubernetes documentation about [Memory resources](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/) and the [CPU resources](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/) for more details.
+
+```yaml
+# yba-values.yaml
+yugaware:
+  # other values…
+  resources:
+    requests:
+      cpu: "4"
+      memory: "5Gi"
+    # optionally set limits
+    limits:
+      cpu: "5"
+      memory: "8Gi"
+```
+
+Similarly, you can modify the values for Prometheus and Postgres containers which are part of the chart.
+
+```yaml
+# yba-values.yaml
+prometheus:
+  # other values…
+  resources:
+    requests:
+      cpu: "2"
+      memory: "6Gi"
+
+postgres:
+  # other values…
+  resources:
+    requests:
+      cpu: "1.5"
+      memory: "2Gi"
+```
+
+### Run containers as non-root
+
+The Postgres and Nginx containers always run as non-root. To run rest of the containers as non-root, you can set the following values:
+
+```yaml
+securityContext:
+  enabled: true
+```
+
+### Set pod labels and annotations
+
+Kubernetes resources like pods can have extra metadata in the form of labels and annotations. These key value pairs are used by other tools such as Prometheus. You can add extra labels and/or annotations to the YugabyteDB Anywhere pods as follows:
+
+```yaml
+# yba-values.yaml
+yugaware:
+  # other values…
+  pod:
+    annotations:
+      sidecar.istio.io/proxyCPU: "200m"
+    labels:
+      sidecar.istio.io/inject: true
+      prometheus.io/scrape: true
+```
+
+### Set custom storage class
+
+The storage class used by YugabyteDB Anywhere pods can be changed along with the size of the volume by using following values:
```

Comment:
The storage class used by YugabyteDB Anywhere pods can be changed, along with the size of the volume, by using following values: